b2sdk/v1/session.py

```diff
@@ -0,0 +1,48 @@
+######################################################################
+#
+# File: b2sdk/v1/session.py
+#
+# Copyright 2021 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+
+from b2sdk import _v2 as v2
+from .account_info import SqliteAccountInfo
+
+
+class B2Session(v2.B2Session):
```

Comment:
add a comment. Why are we doing this? `minimum_part_size=response['recommendedPartSize'],` is the critical piece, I think, but I'd rather read it from a comment and not from the code

Reply:
you're right, I'm a strong supporter of documentaing reasons for apiver overrides, this one definetely needs a comment.